README.md

```diff
@@ -6,23 +6,36 @@
 
 ## Getting Started
 
-1. Install Node module dependencies via: `npm install`
-1. Use the default configuration in `./config/env/default.js` or override with your own configuration that matches the `NODE_ENV` environment variable by copying the `./config/env/development.template.js` file and renaming it to match the value of `$NODE_ENV`
+### Prereqs
```

Comment:
I made some changes to the Getting Started section including installing mongo from docker(I had some crazy @swontonsoup clustered config and wanted to get back to the simplest install). All the other changes in this file are from prettier formatting